content/docs/w5yi/greeter-script.md

```diff
@@ -0,0 +1,117 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: W5YI VEC Greeter Script
+linktitle: Sample W5YI VE Greeter Script
+type: book
+# date: 2022-01-14T00:00:00
+lastmod: 2022-01-14T00:00:00
+featured: false
+draft: false
+weight: 10
+
+---
+W5YI VE Greeter Script
+v4 04/08/2021
+
+
+_CVE: make sure Zoom is in Gallery view_
+
+_CVE: make sure Security/Allow participants to share screen is enabled._
+
+_CVE: start examtools session_
+
+
+
+Before we begin, some additional reminders:
+1. Make sure you have your photo ID handy.
+2. Ensure you are 100% focused during the entire exam session.
+3. Any distractions or anomalies could be cause to void the exam at the sole discretion of the VE Team. Anomalies include, but are not limited to:
+   1. Notable distraction by you, such as eyes not remaining on the exam and/or wandering around the room, or any noises.
+   2. Your appearing to reference or use unauthorized materials (e.g. looking at cell phone, even just the lock screen, changing windows on computer, etc.)
+   3. Another person entering the exam space (on camera, physically, or audibly).
+   4. Your getting up or leaving the room during the exam.
+4. Once you start the exam, the Volunteer Examiners will turn off their audio so as to not distract you – but we’ll be listening and watching. Please leave your audio and video on. Give us a shout if you need anything and when you are ready for grading click the "grade exam" button and let us know you are ready.
+5. Make sure to answer all the questions: Only correct answers add points; incorrect answers don't subtract, so it's best to guess rather than leave an answer blank. Good luck with the exam
+
+
+W5YI VE SCRIPT 1
+* Welcome
+* [If the candidate has multiple devices, try to admit the computer first, and the phone next. Instruct the candidate to join
+with video only, not audio, on the phone. This is not always possible, because devices in the waiting room will not
+always be identifiable by candidate name or device type.]
+* Photo ID.
+o Is ID acceptable to all VE’s?
```

Comment:
Presumably these `o` characters here and below are meant to be sub-bullets? You'll want to use asterisk indented.
```suggestion
  * Is ID acceptable to all VE’s?
```